packages/@coorpacademy-components/src/molecule/dashboard/cards-list/test/fixtures/learning-priority-card-list.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Title from '../../../../../atom/title';
+import TitleProps from '../../../../../atom/title/test/fixtures/form-group-title-icon-tag';
+
+import {defaultProps as FirstCard} from '../../../../learning-priority-card/test/fixtures/default';
+import {defaultProps as SecondCard} from '../../../../learning-priority-card/test/fixtures/certification-type-100';
+import {defaultProps as ThirdCard} from '../../../../learning-priority-card/test/fixtures/skill-type-0';
+
+export default {
+  props: {
+    title: <Title {...TitleProps.props} />,
+    subtitle: 'Your top learning priorities defined by your company',
+    cards: [
+      FirstCard,
+      SecondCard,
+      ThirdCard,
+      FirstCard,
+      SecondCard,
+      ThirdCard,
+      FirstCard,
+      SecondCard,
+      ThirdCard
+    ],
+    onScroll: (skip, limit) => {
+      console.log(skip, limit);
+    },
+    type: 'cards'
```

Comment:
I don't think there are a type props defined in `CardsList` component

Reply:
yes, it's a very tricky thing but it is used in the Higher Order Component which render the section, template/common/dashboard/index.js

in order to render the correct section, it uses the property type of the section, see:
````
const Dashboard = props => {
  const {
    sections = [],
    hero,
    welcome,
    cookie,
    popinWithCards,
    'arrows-aria-label': showMoreOnLeftOrRightAriaLabel
  } = props;
  const buildSectionComponent = section => {
    const {type} = section;
    switch (type) {
      case 'hero':
        return <Hero hero={hero} welcome={welcome} />;
      case 'battleRequests':
        return <BattleRequestList {...section} />;
      case 'cards':
        return <CardsList {...section} arrows-aria-label={showMoreOnLeftOrRightAriaLabel} />;
      case 'news':
        return <NewsList {...section} />;
      case 'battle':
        return <StartBattle {...section} />;
      case 'review':
        return <ReviewBanner {...section} />;
      case 'learningProfile':
        return <LearningProfileBanner {...section} />;
      default:
        return null;
    }
  };
````

it gave me some head headache 😅

Reply:
I update the propTypes documentation of cards list component to mention the type and key propTypes

Reply:
Ah, I think the confused was because how `sections` was define in `dashboard` template, I think it should have been defined simi,ar to this: 
https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-components/src/template/back-office/brand-update/index.js#L281

Do you think it would make sense if we  adjust the logic like this since we now have the `type` props in `CardsList`?

`packages/@coorpacademy-components/src/template/common/dashboard/index.js`

> const Dashboard = props => {
>   const {
>     sections = [],
>     hero,
>     welcome,
>     cookie,
>     popinWithCards,
>     'arrows-aria-label': showMoreOnLeftOrRightAriaLabel
>   } = props;
>   const buildSectionComponent = section => {
>     const {type} = section;
>     switch (type) {
>       ...
>       case 'cards':
>         return <CardsList {...section} arrows-aria-label={showMoreOnLeftOrRightAriaLabel} />;
>       case 'learningPriorities':
>         return <CardsList {...section} type={type} arrows-aria-label={showMoreOnLeftOrRightAriaLabel} />;
>      ...
>       default:
>         return null;
>     }
>   };

> 

 `packages/@coorpacademy-components/src/molecule/dashboard/cards-list/index.js`

> class CardsList extends React.PureComponent {
> 
>   static propTypes = {
>     ...,
>     type: PropTypes.oneOf(['cards', 'learningPriorities']),
>  };

Reply:
ah yes, maybe I didn't pushed yet when you review the PR but it's what I've done in in the template/common/dashboard now